src/System.Net.Sockets.Legacy/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -51,6 +51,7 @@ public SendPacketsAsync(ITestOutputHelper output)
         #region Basic Arguments
 
         [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
I noticed that none of the tests in this file apply to non-Windows platforms. It would be interesting to understand why: maybe a TODO item to add Unix tests that clarify the behavior?